compiler/plugins/target/LLVMCPU/LLVMTargetOptions.cpp

```diff
@@ -279,9 +266,17 @@ LLVMTarget::loadFromConfigAttr(Location loc, DictionaryAttr config,
     target.copy(defaultTarget);
   }
 
-  target.dataLayout = getString("data_layout", DEFAULT_DATA_LAYOUT, false);
-  target.vectorWidthInBytes =
-      getInt64("native_vector_size", DEFAULT_VECTOR_WIDTH_IN_BYTES);
+  if (std::optional<StringRef> configDataLayout = getConfigDataLayout(config)) {
```

Comment:
Can simplify to
```suggestion
target.dataLayout = getConfigDataLayout(config).value_or(DEFAULT_DATA_LAYOUT);
```
Same below.